framework/include/restart/RestartableData.h

```diff
@@ -90,7 +98,7 @@ class RestartableData : public RestartableDataValue
    * @returns a read-only reference to the parameter value.
    */
   T & get() { return *_value_ptr; }
-  // const T & get() const { return *_value_ptr; } // TODO: This should be used; above deprecated
+  const T & get() const { return *_value_ptr; } // TODO: This should be used; above deprecated
```

Comment:
Should probably add an issue to go through and move code over if possible

Reply:
I added a deprecated message and updated MOOSE to use the correct methods.